great_expectations/execution_engine/pandas_execution_engine.py

```diff
@@ -626,7 +682,7 @@ def _split_on_hashed_column(
         """Split on the hashed value of the named column"""
         try:
             hash_method = getattr(hashlib, hash_function_name)
-        except (TypeError, AttributeError) as e:
+        except (TypeError, AttributeError):
```

Comment:
We don't actually use the value of `e` so I've removed it from any applicable `except` clauses

Reply:
❤️